scripts/examples/gn_silabs_example.sh

```diff
@@ -227,6 +227,8 @@ else
                 ;;
             --docker)
                 optArgs+="efr32_sdk_root=\"$GSDK_ROOT\" "
+                optArgs+="wiseconnect_sdk_root=\"$WISECONNECT_SDK_ROOT\" "
+                optArgs+="wiseconnect_sdk_root=\"$WISECONNECT_SDK_ROOT\" "
```

Comment:
I think you copied twice the same by mistake and the second one should be wifi_sdk no?